src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Inflector;
+
+/**
+ * Converts words between singular and plural forms.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+final class Inflector
```

Comment:
You're right. To avoid the problem with an incomplete interface, maybe can we introduce a `PluralizerInterface` (and later we'll add a `SingularizerInterface`).

That way, there is no need to mark the component `@internal`.